beekeeper-cleanup/pom.xml

```diff
@@ -86,6 +92,12 @@
       <version>${s3mock.version}</version>
       <scope>test</scope>
     </dependency>
+<!--    <dependency>-->
```

Comment:
Not at all - this is a net new dependency that I had commented it out as I was hitting some issues with integration tests due to java version incompatibility. 

This may be a good point to bring up a question I had on what the target Java version is for beekeeper. From what I could gather from the pom / docker image, beekeeper was targeted at Java 11, but the HMS library seems to be limited to Java 7/8. 

If that assumption / doc is correct, are there any alternatives to that library that could be used to accomplish the table deletes in hive? or would we need to refactor beekeeper to be compliant with Java 8?

Reply:
Good question. So... we wrote Beekeeper using Java 11 as initially it didn't have any Hive dependencies. With a related project we ran into the issue you describe with some code in Hive that isn't compatible with Java >=9. We raised a PR against Hive which has been merged into the necessary branches, we're just waiting for a Hive 2.3.7 release with those changes in it. We have an internally patched version that we've been using up to now but that's obviously not ideal and will be problematic for a production release. We should reach out to the Hive community and see if we can convince them to do a release which we can then use.

Reply:
Ok awesome - glad that we have a way forward with it. If we aren't able to get a release for it anytime soon, I can use the internal version in the meantime to get the feature up and running. Is there a specific route that we usually go in reaching out to the Hive maintainers? or is it just through their Jira?

Reply:
I commented on a recent email thread asking about a 2.4.0 release (see http://mail-archives.apache.org/mod_mbox/hive-user/201912.mbox/%3CCAE-i9zvR9s26DAz14Ywzuo99vUcGq_gtBHO83TuMV0vaw_8LyQ%40mail.gmail.com%3E) but I don't know if that will go anywhere. If you want to try help here please add a comment on https://issues.apache.org/jira/browse/HIVE-21508 asking when this will go into a Hive 2.3 release and then also put a post on the user email list referencing the Jira and asking about a release etc. The more people that ask the more likely it is they will do it.